### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]:NVDA and Jaws are not announcing the content present in " Run on Web App for containers" Dialog sequentially .

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1514]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app.
4. "Run on web app containers" dialog will open.
5. In Run on web App containers dialog, Press "Insert+B" to read  the entire dialog then observe the NVDA and JAWS Behavior

**Expected Result:**
NVDA and Jaws Should announce in the following manner

- Run on Web App for Containers  dialog  
- Name:  edit  alt+N
- Web App for Containers: sample:sample
- Share  check box  not checked  alt+S
- Single instance only  check box  not checked  alt+I
- Azure Container Registry
- Docker File
edit
Browse  button  Shift+Enter
- Container Registry
combo box  collapsed
- Server URL
-  Username
- Password
- Image and tag Startup File
-  Web App for Containers
   Use Existing Web App  radio button  checked  2 of 2
 Create New Web App  radio button  not checked  1 of 2
- Refresh  button  Refresh
- table
- Artifact
  Artifact Combo box
- Before launch: Build Artifacts, Activate tool window
Add (Alt+Insert)  button  Add (Alt+Insert)
Remove (Alt+Delete)  button  Remove (Alt+Delete)
Edit (Enter)  button  Edit (Enter)
Up (Alt+Up)  button  Up (Alt+Up)
Down (Alt+Down)  button  Down (Alt+Down)
list
Build 'unnamed' artifact  1 of 1
- Show this page  check box  not checked
- Activate tool window  check box  checked
-   Invalid Data:  Please specify a valid docker file.  
- button  Show help contents
- Run  button
- Cancel  button
- Apply  button  alt+A

**Actual Result:**
**NVDA Behavior**: NVDA is announcing as
Run on Web App for Containers  dialog  
Invalid Data: Please specify a valid docker file.
Artifact
Azure Container Registry
Docker File
Server URL
Username
Password
Image and tag
Container Registry
Startup File
 /
Web App for Containers
Before launch: Build Artifacts, Activate tool window
   Invalid Data:  Please specify a valid docker file.  
separator
Artifact
combo box  collapsed
Azure Container Registry
separator
Docker File
edit
blank
Browse  button  Shift+Enter
Server URL
edit
blank
Username
edit
blank
Password
password edit  editable
blank
Image and tag
edit
blank
Container Registry
combo box  collapsed
Startup File
edit
blank
 /
Web App for Containers
separator
table
Name
Resource Group
Refresh  button  Refresh
Use Existing Web App  radio button  checked  2 of 2
Create New Web App  radio button  not checked  1 of 2
Before launch: Build Artifacts, Activate tool window  alt+B
separator
list
Build 'unnamed' artifact  1 of 1
Add (Alt+Insert)  button  Add (Alt+Insert)
Remove (Alt+Delete)  button  Remove (Alt+Delete)
Edit (Enter)  button  Edit (Enter)
Up (Alt+Up)  button  Up (Alt+Up)
Down (Alt+Down)  button  Down (Alt+Down)
Show this page  check box  not checked
Activate tool window  check box  checked
Name:  alt+N
Name:  edit  alt+N
Web App for Containers: sample:sample
Share  check box  not checked  alt+S
Single instance only  check box  not checked  alt+I
button  Show help contents
filler
Run  button
Cancel  button
Apply  button  alt+A
**Jaws Behavior:**
Run on Web App for Containers dialog
Artifact Azure Container Registry Docker File Server URL Username Password Image and tag Container Registry Startup File wiprosample1.azurecr.io/ Web App for Containers App name .azurewebsites.net Subscription Resource Group App Service Plan Location Pricing Tier Location Pricing Tier N/A N/A Before launch: Build Artifacts, Activate tool window 
Apply Button

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![hierarcy is incorrect for run on web app for containers window](https://user-images.githubusercontent.com/22956130/36150425-ae13d30e-10e9-11e8-886a-48a6533e5d09.jpg)
![run on web app for containers-insert-b 2](https://user-images.githubusercontent.com/22956130/36150462-d000a992-10e9-11e8-8f0e-145ae9acaee7.jpg)

**Comment by Eskibear** [comment]
duplicate with #1516

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749981
